### PR TITLE
[GEN-1503, GEN-230] Add nextflow workflow for clinical release

### PR DESCRIPTION
**Pull request by thomasyu888**
The clinical release was missing a nextflow workflow.  This creates the docker image and a thin wrapper around the script so it is easier to run.
image

Here's an example execution: https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/genie-bpc-project/watch/2OZvdTNygSf9Pd

Reminder to update the genie-bpc-pipelines on Seqera platform when this is merged
* Update revision for BPC clinical
* Update entry for genie-bpc-pipeline

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -104,3 +105,7 @@ workflow {
     THE END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
+
+workflow CLINICAL_RELEASE {
```
Comment (rxu17) [inline main.nf:109]: When do we run the clinical release?

**Review by rxu17 — APPROVED** [review]
LGTM! Just a few questions

File: modules/run_clinical_release.nf [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+   if (production) {
+      """
+      cd /usr/local/src/myscripts/
+      Rscript create_release_files.R -c $cohort -v -s
+      """
+   }
+   else {
+      """
+      cd /usr/local/src/myscripts/
+      Rscript create_release_files.R -c $cohort -v
```
Comment (rxu17) [inline modules/run_clinical_release.nf:25]: I also noticed there were other release related scripts in the `release` directory, any reason those are not being run?

File: scripts/release/Dockerfile
Comment (rxu17) [inline scripts/release/Dockerfile:1]: Is this dockerfile meant for just the `create_release_files.R` script or covers all of the scripts in the `release` directory and their dependencies to be run?

File: main.nf (hunk 1 above)
Comment (thomasyu888) [reply main.nf:109]: https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2621931648/GENIE+BPC+Pipeline+SOP#5.-Processing:-generate-clinical-files

Comment (thomasyu888) [reply main.nf:109]: Potentially we combine this with cbioportal export so the releases are run together

File: modules/run_clinical_release.nf (hunk 2 above)
Comment (thomasyu888) [reply modules/run_clinical_release.nf:25]: Needs comment from @Chelsea-Na to see what other scripts should be added or nextflow processes

File: scripts/release/Dockerfile
Comment (thomasyu888) [reply scripts/release/Dockerfile:1]: Just wanted to follow the SOP - probably the other major script is copy_release_files but Im not sure without comment from chelsea

**Pull request merged**